appium-tests/android/android.spec.js

```diff
@@ -21,10 +21,13 @@
  *
 */
 
-// these tests are meant to be executed by Cordova Medic Appium runner
-// you can find it here: https://github.com/apache/cordova-medic/
+// these tests are meant to be executed by Cordova ParaMedic Appium runner
+// you can find it here: https://github.com/apache/cordova-paramedic/
 // it is not necessary to do a full CI setup to run these tests
-// just run "node cordova-medic/medic/medic.js appium --platform android --plugins cordova-plugin-camera"
+// for android 5.1, run:
+//      node cordova-paramedic/main.js --platform android --plugin cordova-plugin-camera --skipMainTests --saucePlatformVersion "5.1" --target api22
```

Comment:
I still think that the user should know better which emulator he wants to use in the test run. For example, one of the emulator may not have camera enabled. I doubt that Appium would know that when selecting "best" emulator.
Also please don't forget that `--target` arg specifies not only emulator for Appium tests, but also for our regular "mobilespec" ones. It just seems natural to me that the Appium should use already opened and ready-for-action emulator instead of choosing another one.
I'm not against any changes in paramedic, just letting you know how it works right now. 😉